plugins/less.js

```diff
@@ -26,25 +26,24 @@ module.exports = function (options) {
     options.ext = options.ext || 'less';
 
     return function (data, args, callback) {
-        var parser = new(lib.Parser)({
+        var options = {
             paths: args.paths,
             filename: args.context.name,
             dumpLineNumbers: 'comments'
-        });
+        };
 
         try {
```

Comment:
The function internally could throw an error and one of the tests even checks for this case to occur.